<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPost</class>
 <widget class="QWidget" name="AddPost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="postChooseSkills">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Вибрати вміння</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Опис посади:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountOfpalces">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="postMaxSalary">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Назва посади:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="postName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальна зарплата:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="postDescription">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>241</width>
     <height>78</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Мінімальна зарплата:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="postMinSalary">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Вміння:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPost">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Зберегти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearPostField">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистити</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Вибрані вміння:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кількість вільних місць:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
